src/hooks/index.ts

```diff
@@ -105,6 +105,7 @@ export const useWallet = (wallet: YoroiWallet, event: WalletEvent['type']) => {
 
 export const useReceiveAddresses = (wallet: YoroiWallet) => {
   useWallet(wallet, 'addresses')
+  useWallet(wallet, 'transactions')
```

Comment:
Subscribing the hook to the event `transactions`, so the addresses are synced when the transaction cache updates.